index.html

```diff
@@ -1,30 +1,53 @@
-<!DOCTYPE html>
-<html lang="en">
-
-<head>
+ <!DOCTYPE html>
+ <html lang="en">
+ <head>
 	<meta charset="UTF-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=adge">
-
-	<title>Document</title>
-=======
-	<title>Паста Бар</title>
-
-    <link rel="stylesheet" href="style.css">
+   	<meta http-equiv="X-UA-Compatible" content="ie=adge">
+	<title>PastaBar</title>
+	<link rel="stylesheet" href="style.css">
 	<script src="script.js">
-
 	</script>
-</head>
-
-<body>
-
- <em><h1> Паста Бар </h1></em>
-
-=======
-
-
- <h1><i> Паста Бар </i></h1>
- 
-</body>
- 
-</html>
\ No newline at end of file
+ </head>
+ <body>
+ <header class="header">
+	 <div class="container">
+		 <div class="header_iner">
+			 <div class="header_logo">PasataBar</div>
+			 <nav class="nav">
+				<a class="nav_link active" href="#"> Меню </a>
+				<a class="nav_link" href="#">Винная карта</a>
+				<a class="nav_link" href="#">Виды Паст</a>
+				<a class="nav_link" href="#">Доставка</a>
+				<a class="nav_link href="#">Отзывы</a>
+				<a class="nav_link href="#">Контакты</a>
+			 </nav>
+		 </div>
+	 </div>
+ </header>
+	 <div class="intro">
+		<div class="container">
+			<div class="intro_iner">
+			<h1 class="intro_title">PastaBAr</h1>
+			<a class="btn" href="#">Заказать</a>
+			</div>
+		</div>
+	 </div>
+
+<section class="section">
+	<div class="container">
+		<div class="section_header">
+			<h3 class="section_suptitle">Меню</h3>
+			<h2 class="section_title">Пасты</h2>
+			<div class="section_text">
+				<p> паста с курицой и брокколи
+					куриное филе, брокколи, чери, сливки, бульон, сыр. 
+				
+				</p>
+			</div>
+		</div>
+	</div>
+</section>
+
```

Comment:
Why you didn't declare here footer area?

Reply:
because it was a sketch, in the futer we will fix everything